<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreWindow</class>
 <widget class="QMainWindow" name="CoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoreWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="file_navigation">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_test_project">
       <attribute name="title">
        <string>Проекты тестирования</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_create_new">
              <property name="text">
               <string>Создать новый
проект</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_open_redactor">
              <property name="text">
               <string>Открыть для 
редактирования</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_update_project">
              <property name="text">
               <string>Обновить список</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_directory_project">
              <property name="text">
               <string>Открыть директорию
с проектами</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="listWidget_project">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_project">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000000</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>1</number>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>24</number>
            </attribute>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Название</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Создатель</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Дата создания</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Кол-во вопросов</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_result_record">
       <attribute name="title">
        <string>Результаты тестирования</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListWidget" name="listWidget_result">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget_result">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>150</number>
              </attribute>
              <row>
               <property name="text">
                <string/>
               </property>
              </row>
              <column>
               <property name="text">
                <string>Название</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Тестируемый</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Кол-во баллов</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Оценка</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_update_record">
              <property name="text">
               <string>Обновить список</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_directory_result">
              <property name="text">
               <string>Открыть директорию
с результатами</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Тестирование</string>
    </property>
    <addaction name="action_start_testing"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about_program"/>
   </widget>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_start_testing">
   <property name="text">
    <string>Начало нового тестирования...</string>
   </property>
  </action>
  <action name="action_view_testing_record">
   <property name="text">
    <string>Просмотр записей тестирований...</string>
   </property>
  </action>
  <action name="action_about_program">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
